.github/workflows/rust-fmt.yml

```diff
@@ -0,0 +1,35 @@
+name:                         Rustfmt
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - stable
+
+jobs:
+  rustfmt:
+    name:                     Cargo fmt
```

Comment:
Especially for formatting, I'd recommend to bind the version and update manually and explcitily.

Reply:
OS version or toolchain version?

Reply:
Toolchain version. OS version should not influence the behavior of a set toolchain version.